### PR TITLE
Charts Init

### LINKED ISSUE #7: Initialize charts
Create 3 charts components using chartjs, follow the theme we have on the design.
- [x] normal chart component
- [x] longshort chart component
- [x] returns at expiry chart component

### PR DESCRIPTION
**Pull request by ethedev**

### CODE REVIEW

**Comment by LePashtet** [comment]
@ethedev Now builds properly

File: src/App.vue [hunk 1]
```diff
@@ -1,4 +1,10 @@
 <template>
+  <div id="nav">
+    <router-link to="/">Home</router-link> |
+    <router-link to="/explore">Explore</router-link>
+    <router-link to="/insidebar">InSidebar</router-link>
+    <router-link to="/markets">Markets</router-link>
+  </div>
```
Comment (ethedev) [inline src/App.vue]: thats no longer needed, from now any new page needs to be added inside the header component
```suggestion

```

File: src/components/charts/LineChart.vue [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+    this.chartData.forEach(item => {
+      data.datasets.push(item)
+    })
+
+    canvas.height = container.clientHeight;
+    canvas.width = container.clientWidth;
+
+
+    let myChart = new Chart(ctx, {
```
Comment (ethedev) [inline src/components/charts/LineChart.vue:95]: some documentation would be great here, specially to what we need to pass inside and how it needs to look like

File: src/components/charts/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Chart implementation for Synths
```
Comment (ethedev) [inline src/components/charts/readme.md]: better format the code blocks

File: src/components/charts/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+borderWidth: 3,
+
+tension: 0.4,
+
+fill: false,
+
+borderColor: '#FFFFFF',
+
+data: [12, 25, 1, 5, 20, 3]
+}]
```
Comment (ethedev) [inline src/components/charts/readme.md]:
```suggestion
# Chart implementation for Synths
LineChart extends Chart.js/LineChart

## Props:
	- labels: Array of ?
	- chartData: Array of objects

## Example:
```
labels: [12, 25, 1, 5, 20, 3],
chartData: [
  {
    backgroundColor: '#FFFFFF',
    borderWidth: 3,
    tension: 0.4,
    fill: false,
    borderColor: '#FFFFFF',
    data: [
      12,
      25,
      1,
      5,
      20,
      3
    ]
  }
]
```
```

**Pull request merged**